xarray/core/options.py

```diff
@@ -1,11 +1,37 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
+from .pycompat import basestring
 
+# Check which of the backend I/O engine packages are installed
+_AVAILABLE_IO_ENGINES = []
+try:
+    import netCDF4  # noqa
+    _AVAILABLE_IO_ENGINES.append('netcdf4')
+except ImportError:
+    pass
+try:
+    import pydap  # noqa
+    _AVAILABLE_IO_ENGINES.append('pydap')
+except ImportError:
+    pass
+try:
+    import scipy.io.netcdf  # noqa
+    _AVAILABLE_IO_ENGINES.append('scipy')
+except ImportError:
+    pass
+try:
+    import h5netcdf.legacyapi  # noqa
+    _AVAILABLE_IO_ENGINES.append('h5netcdf')
+except ImportError:
+    pass
+if len(_AVAILABLE_IO_ENGINES) == 0:
+    raise RuntimeError('No suitable I/O engines found installed')
```

Comment:
None of these backends are required for using xarray, so this error will need to get raised when opening/saving a file instead, not merely when importing `xarray.core.options`.

Reply:
This exception will be raised only when none of the known I/O engines can be imported, netCDF4 included. I assume netCDF4 is the only one required so I think it is safe to keep it as is.

Reply:
Not even netCDF4 is a required dependency. We only require numpy and pandas: http://xarray.pydata.org/en/stable/installing.html

Reply:
Obviously, you need an I/O engine to read/write files. But not to import xarray.